bzip2/PSPBUILD

```diff
@@ -0,0 +1,37 @@
+# Maintainer: Wouter Wijsman <wwijsman@live.nl>
+pkgname=bzip2
+pkgver=1.0.6
+pkgrel=1
+pkgdesc="A high-quality data compression library"
+arch=('mips')
+url="http://www.sourceware.org/bzip2/"
+license=('custom')
+depends=()
+makedepends=()
+optdepends=()
+source=(
+	"ftp://sourceware.org/pub/${pkgname}/${pkgname}-${pkgver}.tar.gz"
+	"${pkgname}-${pkgver}-PSP.patch"
+)
+sha256sums=(
+	'a2848f34fcd5d6cf47def00461fcb528a0484d8edef8208d6d2e2909dc61d9cd'
+	'SKIP'
+)
+
+prepare() {
+	cd "$pkgname-$pkgver"
+	patch -p1 < ../${pkgname}-${pkgver}-PSP.patch
+}
+
+build() {
+	cd "$pkgname-$pkgver"
+	make -j`nproc`
```

Comment:
This is wrong. You only ever want to set that to `-j1`, if the Makefile does not support parallel builds. This is set globally in `makepkg.conf`.